Added info/resource for special variables in Hello World tutorial (#336)
CHTC/chtc-website-source: uw-research-computing/r-jobs.md

```
{:.term}

In the R terminal, install your packages using `install.packages`.

``` 
> install.packages("package_name")
```
{:.term}

Replace "package_name" with the name of the package you wish to install. 

The first time you will be prompted to choose a \"CRAN mirror\" - this

> is where R is downloading the package. Choose any option that does **not** state `https`
> by the name as we only want locations with `http` configuration. 

Reviewer: ```suggestion
is where R is downloading the package. Choose any US-based location to download.  
```
